src/Core/Framework/MessageQueue/ScheduledTask/SymfonyBridge/ScheduleProvider.php

```diff
@@ -13,7 +13,7 @@
 use Symfony\Contracts\Cache\CacheInterface;
 
 /**
- * @experimental stableVersion:v6.7.0 feature:SYMFONY_SCHEDULER
+ * @experimental stableVersion:v6.8.0 feature:SYMFONY_SCHEDULER
```

Comment:
i'm not sure if we should make this stable 🤔 
there were some issues in the past, therefore I propose to move it to the 6.8